test/functional/index.test.js

```diff
@@ -1354,4 +1356,101 @@ describe('Indexes', function() {
       });
     }
   });
+
+  context('should throw an error if commitQuorum specified on MongoDB < 4.4', function() {
+    function throwErrorTest(testCommand) {
```

Comment:
This seems like something that we might want to use in the future and worth making more generic and putting in `test/functional/shared.js`.

Reply:
Agreed we should pull it out if we need it down the line. These helpers can be nice, but they can also be a pain if you have to change a test down the line in a way the helper isn't designed to support, so I'm hesitant about adding them preemptively.